php/media/class-gallery.php

```diff
@@ -398,6 +396,39 @@ protected function register_settings() {
 		$media_settings->add_setting( $this->settings );
 	}
 
+	/**
+	 * Inits the cloudinary gallery using block attributes.
+	 *
+	 * @param string $content The post content.
+	 * @param array  $block   Block data.
+	 *
+	 * @return string
+	 */
+	public function prepare_block_render( $content, $block ) {
+		if ( 'cloudinary/gallery' !== $block['blockName'] ) {
+			return $content;
+		}
+
+		$attributes                = Utils::expand_dot_notation( $block['attrs'], '_' );
+		$attributes                = array_merge( self::$default_config, $attributes );
+		$attributes['mediaAssets'] = $attributes['selectedImages'];
+		$attributes['cloudName']   = $this->media->plugin->components['connect']->get_cloud_name();
+		unset( $attributes['selectedImages'], $attributes['customSettings'] );
+
+		ob_start();
+		?>
+<script>
+	(function () {
+		const attributes = JSON.parse( '<?php echo wp_json_encode( $attributes ); ?>' );
```

Comment:
What is the purpose of the `JSON.parse()` here? Could we assign the output of `wp_json_encode()` directory?

<img width="283" alt="Screenshot 2021-01-15 at 22 36 53" src="https://user-images.githubusercontent.com/169055/104776093-2d29a100-5782-11eb-8316-8bd06ed555c9.png">

Also, could we use `var` here just for compatibility with older browsers?